### PR TITLE
Update conditionalaccessplatforms.md

### PR DESCRIPTION
**Pull request by SanDeo-MSFT**
Schema update PR: 5414822

### CODE REVIEW

**Comment by SanDeo-MSFT** [comment]
#sign-off

**Review by FaithOmbongi — APPROVED** [review]

File: api-reference/beta/resources/conditionalaccessplatforms.md [hunk 1]
```diff
@@ -19,8 +19,8 @@ Platforms included in and excluded from the policy scope.
 
 | Property     | Type        | Description |
 |:-------------|:------------|:------------|
-|includePlatforms|conditionalAccessDevicePlatform collection| Possible values are: `android`, `iOS`, `windows`, `windowsPhone`, `macOS`, `all`, `unknownFutureValue`.|
-|excludePlatforms|conditionalAccessDevicePlatform collection| Possible values are: `android`, `iOS`, `windows`, `windowsPhone`, `macOS`, `all`, `unknownFutureValue`.|
+|includePlatforms|conditionalAccessDevicePlatform collection| Possible values are: `android`, `iOS`, `windows`, `windowsPhone`, `macOS`, `linux`, `all`, `unknownFutureValue`.|
+|excludePlatforms|conditionalAccessDevicePlatform collection| Possible values are: `android`, `iOS`, `windows`, `windowsPhone`, `macOS`, `linux`, `all`, `unknownFutureValue`.|
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/conditionalaccessplatforms.md:23]: Seen from Aksha's PR that the value `linux `is being added after the sentinel value. This change is required for customer success.
```suggestion
|includePlatforms|conditionalAccessDevicePlatform collection| Possible values are: `android`, `iOS`, `windows`, `windowsPhone`, `macOS`, `all`, `unknownFutureValue`, `linux`. Note that you must use the `Prefer: include-unknown-enum-members` request header to get the following value(s) in this [evolvable enum](/graph/best-practices-concept#handling-future-members-in-evolvable-enumerations): `linux`. |
|excludePlatforms|conditionalAccessDevicePlatform collection| Possible values are: `android`, `iOS`, `windows`, `windowsPhone`, `macOS`, `all`, `unknownFutureValue`, `linux`. Note that you must use the `Prefer: include-unknown-enum-members` request header to get the following value(s) in this [evolvable enum](/graph/best-practices-concept#handling-future-members-in-evolvable-enumerations): `linux`. |
```

**Review by FaithOmbongi — APPROVED** [review]
Approved with suggestions.

File: changelog/Microsoft.IdentityProtectionServices.json [hunk 2]
```diff
@@ -1,5 +1,23 @@
 {
 	"changelog": [
+		{
+      			"ChangeList": [
+        			{
+					  "Id": "8e875dc0-5df2-43c8-8c6b-316aed281493",
+					  "ApiChange": "Member",
+					  "ChangedApiName": "linux",
+					  "ChangeType": "Addition",
+					  "Description": "Added the **linux** member to the **conditionalAccessDevicePlatform** enumeration.",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.IdentityProtectionServices.json:10]: Suggestion to enhance the description and give the customer added context
```suggestion
					  "Description": "Added the **linux** member to the **conditionalAccessDevicePlatform** enumeration. This update allows devices running the Linux platform to be targeted for inclusion or exclusion in conditional access policies.",
```

**Pull request merged**